packages/jaeger-ui/src/components/Monitor/ServicesView/serviceGraph.tsx

```diff
@@ -87,30 +79,20 @@ export class ServiceGraphImpl extends React.PureComponent<TProps> {
 
       this.getData().forEach((line: ServiceMetricsObject, idx: number) => {
         graphs.push(
-          <AreaSeries
+          <Area
             key={i++}
-            data={line.metricPoints ? line.metricPoints : []}
-            getNull={(d: Points) => d.y !== null}
-            onNearestX={(_datapoint: Points, { index }: { index: number }) => {
-              this.setState({
-                crosshairValues: this.getData().map((d: ServiceMetricsObject) => ({
-                  ...d.metricPoints[index],
-                  label: d.quantile,
-                })),
-              });
-            }}
-            opacity={0.1}
-            color={[color || this.colors[idx]]}
+            type="monotone"
+            dataKey="y"
+            connectNulls
+            fillOpacity={0.1}
+            stroke={color || this.colors[idx]}
+            fill={color || this.colors[idx]}
```

Comment:
Aren't we passing on the data prop here?

Reply:
There are no props to pass the data here. Also, we don't need to pass the data props here; the data props is passed to the parent component.